compiler/src/lib.rs

```diff
@@ -58,3 +58,6 @@ pub use self::value::*;
 
 pub mod stage;
 pub use self::stage::*;
+
+pub mod option;
+pub use self::option::*;
```

Comment:
nit: rename to `options` and please order these alphabetically.

p.s. `self::` is no longer necessary.